Rmarkdown/Multiple_Ascending_Dose_PKPD_ordinal.Rmd

```diff
@@ -118,16 +118,17 @@ pd_ordinal_label   = paste0("Ordinal PD Marker (", pd_units, ")")
 
 ```
 
-
 ## Provide an overview of the data
```

Comment:
@margoal1 - Should the red curve (Severe) be able to go to zero?  It looks a bit funny that throughout, if the red curve (Severe) is equal to zero, then it disappears.  It feels like something is wrong.  This issue arises in two stets of plots

Reply:
That's a good question. This would have to be updated in how the xgx_stat_ci function treats ordinal data in the next xgxr update. I suppose the ideal output would be to have the points and lines go to zero, but what about the errorbars?

Reply:
Is there a reason error-bars should be a problem for ordinal but not binary variables?  The error-bars work great on our [binary page](https://opensource.nibr.com/xgx/Multiple_Ascending_Dose_PKPD_binary.html), even for 0 events.  And binom::binom.exact() can report confidence intervals when there are 0 events (see below).  I'll go ahead and file an issue under xgxr.

```
> binom::binom.exact(x = 0, n = 3)
  method x n mean lower     upper
1  exact 0 3    0     0 0.7075982

> binom::binom.exact(x = 0, n = 300)
  method x   n mean lower      upper
1  exact 0 300    0     0 0.01222097
```